src/App.js

```diff
@@ -0,0 +1,43 @@
+import { Provider } from 'react-redux';
+import {
+  BrowserRouter, Route, Routes, Navigate,
+} from 'react-router-dom';
+import './App.css';
+import MyReservations from './pages/my-reservation/MyReservations';
+import DeleteCar from './pages/delete-car/DeleteCar';
+import Navbar from './components/Navbar/Navbar';
+import Reserve from './pages/Reserve/Reserve';
+import AddCar from './pages/AddCar/AddCar';
+import CarList from './components/cars-components/CarList';
+import CarDetails from './components/cars-components/CarDetails';
+import store from './redux/store';
+import ProtectedRoute from './routing/ProtectedRoute';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this function to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

That way, all your functions will be ES6, and comply with the latest standards.  💪

Reply:
Thank you for all the suggestions 🙏